### ISSUE TITLE
{Linked:Bug247271}[Adaptive Card- Designer] - Talkback is announcing incorrect index count while navigating on the dropdown option.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#466]
### Target Platforms

Android

### SDK Version

1.0.1

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: Samsung S24
Android Version: 14
Application: Adaptive Cards Visualizer
Screen Reader: Talkback

**Pre-Requisite:**
Upload the Json from device and activate the Visual tab item.

**Repro Steps:**
1. Run the attached JSON.
2. Perform right swipe and navigate to 'Sample Label' drop down field and double tap to activate it.
3. Navigate the list and observe the issue.

**Actual Result:**
While navigating on the list items present under the 'Sample Label' dropdown, Talkback is announcing the incorrect position as 1 of 5 or 2 of 5 and so on while there are only 4 items are present on the list.
Also, it is not announcing the total number of list items present when landing on the first item.

**Expected Result:**
Talkback should announce correct index count for the list items present under the drop down. It should announce as 1 of 4 or 2 of 4 so on.

**User Impact:**
Screen reader users relying on TalkBack may get confused due to incorrect index count being announced while navigating through the dropdown options. This may lead to misunderstanding about the number of available options or their current position in the list, making it difficult for users to make accurate selections and efficiently navigate the dropdown.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html?utm_source=chatgpt.com

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/b928e660-d638-49f1-8a69-7aaad0f35ae3

### Card JSON

```JSON
{

  "type": "AdaptiveCard",

  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",

  "version": "1.0",

  "body": [

    {

      "type": "TextBlock",

      "text": "Sample text",

      "style": "default",

      "weight": "Bolder",

      "wrap": true

    },

    {

      "type": "ColumnSet",

      "columns": [

        {

          "type": "Column",

          "width": "stretch",

          "items": [

            {

              "type": "Input.ChoiceSet",

              "id": "dropdownSelection",

              "style": "compact",

              "placeholder": "Sample placeholder",

              "value": "",

              "label": "Sample label",

              "isRequired": true,

              "errorMessage": "error message",

              "choices": [

                {

                  "title": "Metro Transit",

                  "value": "metro"

                },

                {

                  "title": "City Bus",

                  "value": "bus"

                },

                {

                  "title": "Regional Rail",

                  "value": "rail"

                },

                {

                  "title": "Airport Shuttle",

                  "value": "shuttle"

                }

              ]

            }

          ]

        }

      ]

    },

    {

      "type": "ColumnSet",

      "columns": [

        {

          "type": "Column",

          "width": "stretch",

          "items": [

            {

              "type": "ActionSet",

              "actions": [

                {

                  "type": "Action.Submit",

                  "data": {

                    "actionSubmitId": "Not_Now"

                  },

                  "title": "Not Now",

                  "associatedInputs": "none",

                  "tooltip": "Not now"

                }

              ]

            }

          ]

        },

        {

          "type": "Column",

          "width": "stretch",

          "items": [

            {

              "type": "ActionSet",

              "actions": [

                {

                  "type": "Action.Submit",

                  "title": "Submit",

                  "style": "default",

                  "data": {

                    "actionSubmitId": "Submit",

                    "selectedOption": "${dropdownSelection}"

                  }

                }

              ]

            }

          ]

        }

      ]

    }

  ]

}
```
